spec/fixtures/event_translations.yml

```
@@ -0,0 +1,6 @@
+#  Copyright (c) 2012-2021, Swiss Canoe. This file is part of
+#  hitobito_skv and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_skv.
+
+
```

vielleicht muss hier noch eintrag simuliert werden...

```suggestion
dummy_de: {}
```

Ich bin aber tatsächlich unsicher, ob das so funktioniert.